### ISSUE TITLE
Not properly redrawing

### ISSUE THREAD

**Issue by Heptite** [hexchat/hexchat#2157]
On Windows 10, HexChat 2.14.1, if I switch away from the window and come back, it fails to redraw.  Various elements will redraw if move my mouse around, or switch to another using hotkeys, but nothing entirely comes back.

![hexchatredrawissue](https://user-images.githubusercontent.com/331520/38043812-55feb564-3275-11e8-936d-6f6efc729f67.png)

**Comment by TingPing** [comment]
Using a custom theme or anything?

**Comment by Heptite** [comment]
> Using a custom theme or anything?

No. I updated HexChat this morning (I wish I could recall what my previous version was) and the problem appeared.
